cmd/mt-whisper-importer-writer/verify.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+
+# this is a tool to verify if the data imported via the whisper importer is
+# actually correct. it uses two inputs, the output of whisper-dump.py and the
+# output of mt-store-cat.
+#
+# pipe the whisper-dump.py output into a file:
+# $> whisper-dump.py storage/whisper/some/id/of/a/metric/1.wsp  > /tmp/whisper_dump
+#
+# pipe the mt-store-cat output into another file:
+# $> mt-store-cat -print-ts -from 1487764800 -to 1487785801 normal id 1.d588ebb28e4d2ca64d828cb1eb6066d4 5h > /tmp/mt_store_dump
+#
+# compare the two:
+# $> ./verify.py /tmp/whisper_dump /tmp/mt_store_dump
+#
+#
+# averaged aggregations are a special case. the whisper importer utility
+# converts avg archives into a sum and a cnt archive and inserts them as such.
+# the cnt archive only gets the aggregation span at each timestamp, the sum
+# archive gets the value from whisper * aggregation span, out of these two
+# series metrictank calculcates the average on the fly when it's queried.
+# so to verify the data we need to use the multiplicator parameter, let's say
```

Comment:
We should add the following to make this clearer:
* mt-store-cat only dumps raw series. such as a sum series or a cnt series corresponding to a given metric. it cannot return an avg series like metrictank does. (btw in your tests did you use the `key_cnt_span` syntax as key name, and it worked? I'm not sure if i ever tried that, cool if it worked for you)
* given this limitation, the closest we can get to validating an average, is getting mt-store-cat to dump the sum series, but divide it by the count, to turn it into an average again
* since mt-store-cat has no way to divide, we pass a number as argument to this script and divide in this script
* the number passed is actually a divider, not a multiplicator ?